19/mridubhatnagar/token.json

```diff
@@ -0,0 +1 @@
+{"revoke_uri": "https://accounts.google.com/o/oauth2/revoke", "client_id": "284369609997-5vuiecpa4j3s7sm06i39kaf1sausob1o.apps.googleusercontent.com", "token_expiry": "2018-08-07T10:41:43Z", "token_response": {"expires_in": 3600, "token_type": "Bearer", "access_token": "ya29.GlvyBag1aD5Nv2XWqxH-2rl3LW6Q8_iQWlR3CxXcHB4j-DFE2msAnJM1jUOeYl_740bs2BNpOMbcpS6ABfL2-vc_5X9XuNR8EimgPs-yGTp84uzPEVUcMmMC5dxs", "scope": "https://www.googleapis.com/auth/calendar"}, "id_token_jwt": null, "invalid": false, "id_token": null, "access_token": "ya29.GlvyBag1aD5Nv2XWqxH-2rl3LW6Q8_iQWlR3CxXcHB4j-DFE2msAnJM1jUOeYl_740bs2BNpOMbcpS6ABfL2-vc_5X9XuNR8EimgPs-yGTp84uzPEVUcMmMC5dxs", "token_uri": "https://www.googleapis.com/oauth2/v3/token", "scopes": ["https://www.googleapis.com/auth/calendar"], "_module": "oauth2client.client", "user_agent": null, "client_secret": "m27Yn1-l_i2x01aMncLA1ram", "_class": "OAuth2Credentials", "refresh_token": "1/_nk7lrT-SuTzxZgbRc2KpDU-CJNonuO6IrBJVj8tZBo", "token_info_uri": "https://www.googleapis.com/oauth2/v3/tokeninfo"}
```

Comment:
hope this is a fake token? you probably want to tuck that away in a env variable